### PR TITLE
Create list.css

### PR DESCRIPTION
**Pull request by Radheshyam1234**
Created 4 types of list:
1. List without bullets
2. ordered list
3. Unordered list
4. Spaced list
list-screenshot

### CODE REVIEW

File: Components/list.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.list-container {
+  border-radius: 0.3rem;
+  padding: 1rem 32px 1rem 40px;
```
Comment (Hamdrive) [inline Components/list.css]: Mixing two units is not good practice, and for values greater than 10px prefer to use `rems`

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Update your padding style to use `rems`
Rest all code looks good

File: Components/list.css (hunk 1 above)
Comment (Radheshyam1234) [reply Components/list.css]: I made the changes.

**Review by stutisk — COMMENTED** [review]
code looks great!

**Pull request merged**